vendor/github.com/BurntSushi/toml/.travis.yml

```diff
@@ -2,11 +2,14 @@ language: go
 go:
   - 1.1
   - 1.2
+  - 1.3
+  - 1.4
+  - 1.5
+  - 1.6
```

Comment:
I don't really understand why the go dependency property is a list. I would interpret this as "build this project for every version listed here". In fact There are only single builds in the travis web interface. I would only use the latest go version and that is it.